src/Multipart/AbstractUploader.php

```diff
@@ -147,4 +147,4 @@ protected function getNumberOfParts($partSize)
         }
         return null;
     }
-}
+}
```

Comment:
nit: in [PSR-2 ](https://www.php-fig.org/psr/psr-2/#22-files), it is stated that all files must end in a single blank line, please add that back.  

The reasoning isn't really applicable here, but it's good to follow the standard.